atom/browser/native_window_mac.mm

```diff
@@ -756,6 +756,17 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   return bounds;
 }
 
+void NativeWindowMac::SetSize(const gfx::Size& size, bool animate) {
```

Comment:
All size APIs are directly to the `SetBounds` method, so we only need to do check there.